README.md

```diff
@@ -10,7 +10,9 @@ Refinery is a trace-aware sampling proxy. It collects spans emitted by your appl
 
 ## Setting up Refinery
 
-Refinery is designed to sit within your infrastructure where all sources of Honeycomb events (aka spans if you're doing tracing) can reach it. A standard deployment will have a cluster of servers running Refinery accessible via a load balancer. Refinery instances must be able to communicate with each other to concentrate traces on single servers.
+Refinery is designed to sit within your infrastructure where all sources of Honeycomb events (aka spans if you're doing tracing) can reach it.
+A standard deployment will have a cluster of two or more Refinery processes accessible via a separate load balancer.
+Refinery processes must be able to communicate with each other to concentrate traces on single servers.
```

Comment:
again, using common language, "correct shard for the trace" >> "concentrate traces on single servers"

Reply:
I'm a bit unclear on the call to action for this comment. Are you suggesting something like the following change?

```suggestion
Refinery processes must be able to communicate with each other to shuttle traces to the correct shard.
```

Should we bring up sharding in this introductory paragraph? Currently, Sharding isn't introduced until the architecture section of this README much farther down.